src/code/reducers/modal-dialog.js

```diff
@@ -25,7 +25,8 @@ export default function modalDialog(state = initialState, action) {
     case actionTypes.TOGGLE_MAP:
       return state.merge({
         show: action.isVisible,
-        showMap: action.isVisible
+        showMap: action.isVisible,
+        showAward: false
```

Comment:
Looks like this also sets `showAward` to `false` if the map is getting toggled off. Granted, this probably points more to a problem with the way I designed the way the dialogs get displayed...